core/core.mk

```diff
@@ -94,6 +94,12 @@ libdir = core/lib/libfdt
 include mk/lib.mk
 endif
 
+ifeq ($(CFG_ZLIB),y)
```

Comment:
Right. Making compression support optional is doable of course, but it would make the code a bit harder to read (several #ifdef's in `early_ta.c`).